Lombiq.Hosting.MediaTheme.Deployer/Program.cs

```diff
@@ -49,102 +82,158 @@ private static void HandleParseError(IEnumerable<Error> errors)
         {
             dynamic assetJObject = new JObject();
             assetJObject.SourcePath = Path.Join(
                 MediaThemeAssetsWebPath,
-                assetPath[pathToAssets.Length..].Replace("\\", "/"));
+                assetPath[assetsPath.Length..].Replace("\\", "/"));
             assetJObject.TargetPath = assetJObject.SourcePath;
 
             files.Add(assetJObject);
         }
 
         // Copying assets to deployment directory.
         CopyDirectory(
-            pathToAssets,
+            assetsPath,
             Path.Join(newDirectoryPath, MediaThemeAssetsCopyDirectoryPath),
             areLiquidFiles: false);
 
         // Getting templates.
-        var pathToTemplates = Path.Join(pathToTheme, LocalThemeViewsDirectory);
+        var templatesPath = Path.Join(themePath, LocalThemeViewsDirectory);
 
         var allTemplatesPaths = Directory
-            .EnumerateFiles(pathToTemplates, "*" + LiquidFileExtension, SearchOption.TopDirectoryOnly);
+            .EnumerateFiles(templatesPath, "*" + LiquidFileExtension, SearchOption.TopDirectoryOnly);
 
         foreach (var templatePath in allTemplatesPaths)
         {
             dynamic templateJObject = new JObject();
             templateJObject.SourcePath = Path.Join(
                 MediaThemeTemplatesWebPath,
-                templatePath[pathToTemplates.Length..].Replace("\\", "/"));
+                templatePath[templatesPath.Length..].Replace("\\", "/"));
             templateJObject.TargetPath = templateJObject.SourcePath;
 
             files.Add(templateJObject);
         }
 
         // Copying templates to deployment directory.
         CopyDirectory(
-            pathToTemplates,
+            templatesPath,
             Path.Join(newDirectoryPath, MediaThemeTemplatesCopyDirectoryPath),
             areLiquidFiles: true,
             recursive: false);
 
         dynamic mediaStep = new JObject();
         mediaStep.name = "media";
         mediaStep.Files = files;
+        recipeSteps.Add(mediaStep);
 
-        CreateRecipeAndWriteIt(mediaThemeStep, mediaStep, newDirectoryPath);
+        CreateRecipeAndWriteIt(recipeSteps, newDirectoryPath);
 
         // Zipping the directory.
-        var zippedDirectoryPath = newDirectoryPath + ".zip";
-        ZipFile.CreateFromDirectory(newDirectoryPath, zippedDirectoryPath);
+        var zipFilePath = newDirectoryPath + ".zip";
+        ZipFile.CreateFromDirectory(newDirectoryPath, zipFilePath);
 
         // Getting rid of the original directory.
         Directory.Delete(newDirectoryPath, recursive: true);
 
-        WriteLine("{0} was created successfully. ", zippedDirectoryPath);
+        WriteLine("{0} was created successfully. ", zipFilePath);
+
+        if (string.IsNullOrEmpty(options.RemoteDeploymentUrl))
+        {
+            return;
+        }
+
+        // This is a remote deployment.
+        await RemoteDeploymentHelper.DeployAsync(options, zipFilePath);
```

Comment:
Why not invert the `if`? Also the comment doesn't seem to be useful - the method name already tells us the same.

Reply:
No reason. I initially had more code on this line and wanted an early return before it.